### PR TITLE
specific 

### PR DESCRIPTION
**Pull request by omatrot**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -27,15 +27,7 @@ services:
...
-            - .:/var/www/html
-
-###########################
-# Setup the testing container
-###########################
-    testing:
-        build: ./docker/php/
-        volumes_from:
-            - php
+            - /Users/omatrot/Documents/etp_web:/etp_web
```
Comment (settermjd) [inline docker-compose.yml:30]: I'm guessing that this worked locally. Given the configuration, I wouldn't expect it to work for anyone else, unfortunately.

File: docker-compose.yml [hunk 2]
```diff
@@ -45,9 +37,9 @@ services:
...
             - 3306
         volumes:
-            - database_data:/var/lib/mysql
+            - database_data:/usr/local/var/mysql
         environment:
-            MYSQL_ROOT_PASSWORD: secret
-            MYSQL_DATABASE: project
-            MYSQL_USER: project
-            MYSQL_PASSWORD: project
+            MYSQL_ROOT_PASSWORD: knhGxQHUify7
```
Comment (settermjd) [inline docker-compose.yml:42]: These settings were deliberately kept generic. Ideally they shouldn't be hard-coded values of any kind, rather drawn from environment variables.

File: docker/nginx/default.conf [hunk 3]
```diff
@@ -4,7 +4,7 @@ server {
     listen 80 default_server;
 
     # Set the document root of the project
-    root /PATH/TO/YOUR/DOCUMENT/ROOT;
+    root /etp_web;
```
Comment (settermjd) [inline docker/nginx/default.conf:7]: This line was deliberately written as it was, and should also be drawn from environment variables, to be fair.

File: docker/nginx/default.conf [hunk 4]
```diff
@@ -32,7 +32,7 @@ server {
     client_max_body_size 100m;
 
     # Specify what happens when PHP files are requested
-    location ~ \.php$ {
+    location ~ \.php$|\.html {
```
Comment (settermjd) [inline docker/nginx/default.conf:35]: Now here, I see what you're after. I don't adhere to mixing PHP in HTML anymore. But I appreciate what your use-case is.

File: docker/php/Dockerfile [hunk 5]
```diff
@@ -1,4 +1,5 @@
 FROM php:7.0-fpm
 
-RUN docker-php-ext-install pdo_mysql \
-    && docker-php-ext-install json
+COPY www.conf /usr/local/etc/php-fpm.d/www.conf
```
Comment (settermjd) [inline docker/php/Dockerfile:3]: I'm curious as to why the RUN statements were removed and the extensions were replaced. Can you share your reasoning?

**Review by settermjd — COMMENTED** [review]
@omatrot, thanks for putting in the effort and sharing your input.

**Comment by omatrot** [comment]
This PR was closed by myself. It should not be reviewed, as you've seen yourself, it contains many things specific to my setup.

**Comment by settermjd** [comment]
True. I wasn't meaning to be harsh in what I said, nor in how I said it.

**Pull request closed without merging**